client/src/App.js

```diff
@@ -1,30 +1,28 @@
-import React, { Component } from 'react';
 import './App.css';
-import Project from './Project';
+import React, {Component} from 'react';
```

Comment:
I would kind of have preferred this to be a separate commit, but 🤷‍♂️.

Reply:
this was from the linter, it wants imports to be in a certain order - not sure what about it you'd like to be separate?